scripts/variantstore/wdl/GvsBulkIngestGenomes.wdl

```diff
@@ -14,6 +14,7 @@ workflow GvsBulkIngestGenomes {
         String sample_id_column_name = "sample_id"
         String? vcf_files_column_name
         String? vcf_index_files_column_name
+        String? sample_set_name ## currently we only allow for one sample set at a time
```

Comment:
it's not necessarily called a sample_set...it's <entity>_set
do we see this naming as a problem?

Reply:
Maybe we should discuss this, it sounds a bit strange